### DISCUSSION TITLE
How to get the steam app through Obtainium?

### DISCUSSION THREAD

**Discussion by hristogochev** [ImranR98/Obtainium#1500] [Q&A] [answered]
First of all, thank you for the great app!

I would like to know how to install the steam mobile app, the first one from [here](https://store.steampowered.com/mobile) (not steam link or steam chat). I get that I must paste the source which is the link I just provided and override the source to HTML (because of [this](https://github.com/ImranR98/Obtainium/pull/1248)) but I'm not sure what other settings I need to put. 

Thanks in advance for your help!

**Comment by ypurpl** [comment]
it isnt open source, but they have a direct link here: https://media.steampowered.com/apps/steam-android/steam-3.7.4.apk
try using direct apk link and do some testing

**Comment by DarkCrypt** [comment, marked as answer]
You can add Steam (mobile, steam link, chat) from the list at [apps.obtainium.imranr.dev](https://apps.obtainium.imranr.dev/).

Or you can use this config. It's different from the website above but still works...

[Steam Mobile](https://apps.obtainium.imranr.dev/redirect?r=obtainium://app/%7B%22id%22%3A%22com.valvesoftware.android.steam.community%22%2C%22url%22%3A%22https%3A%2F%2Fstore.steampowered.com%2Fmobile%22%2C%22author%22%3A%22store.steampowered.com%22%2C%22name%22%3A%22Steam%22%2C%22preferredApkIndex%22%3A0%2C%22additionalSettings%22%3A%22%7B%5C%22intermediateLink%5C%22%3A%5B%5D%2C%5C%22customLinkFilterRegex%5C%22%3A%5C%22%5C%22%2C%5C%22filterByLinkText%5C%22%3Afalse%2C%5C%22skipSort%5C%22%3Afalse%2C%5C%22reverseSort%5C%22%3Afalse%2C%5C%22sortByLastLinkSegment%5C%22%3Afalse%2C%5C%22versionExtractWholePage%5C%22%3Afalse%2C%5C%22requestHeader%5C%22%3A%5B%7B%5C%22requestHeader%5C%22%3A%5C%22User-Agent%3A%20Mozilla%2F5.0%20(Linux%3B%20Android%2010%3B%20K)%20AppleWebKit%2F537.36%20(KHTML%2C%20like%20Gecko)%20Chrome%2F114.0.0.0%20Mobile%20Safari%2F537.36%5C%22%7D%5D%2C%5C%22defaultPseudoVersioningMethod%5C%22%3A%5C%22APKLinkHash%5C%22%2C%5C%22trackOnly%5C%22%3Afalse%2C%5C%22versionExtractionRegEx%5C%22%3A%5C%22%5B0-9%5D%2B%5C%5C%5C%5C.%5B0-9%5D%2B%5C%5C%5C%5C.%5B0-9%5D%2B%5C%22%2C%5C%22matchGroupToUse%5C%22%3A%5C%22%5C%22%2C%5C%22versionDetection%5C%22%3Atrue%2C%5C%22useVersionCodeAsOSVersion%5C%22%3Afalse%2C%5C%22apkFilterRegEx%5C%22%3A%5C%22steam-android%5C%22%2C%5C%22invertAPKFilter%5C%22%3Afalse%2C%5C%22autoApkFilterByArch%5C%22%3Atrue%2C%5C%22appName%5C%22%3A%5C%22Steam%20Mobile%5C%22%2C%5C%22shizukuPretendToBeGooglePlay%5C%22%3Afalse%2C%5C%22allowInsecure%5C%22%3Afalse%2C%5C%22exemptFromBackgroundUpdates%5C%22%3Afalse%2C%5C%22skipUpdateNotifications%5C%22%3Afalse%2C%5C%22about%5C%22%3A%5C%22Buy%20games%2C%20protect%20your%20Steam%20account%2C%20and%20get%20the%20latest%20news%20from%20your%20games%20and%20the%20community.%5C%22%2C%5C%22supportFixedAPKURL%5C%22%3Afalse%7D%22%2C%22overrideSource%22%3A%22HTML%22%7D) - install link

Config= Steam Mobile
App Source URL: `https://store.steampowered.com/mobile`
Override Source: `HTML`
Default Pseudo-Versioning Method: `APK Link Hash`
Version String Extraction ReGex: `[0-9]+\.[0-9]+\.[0-9]+`
Filter APKs by Regular Expression: `steam-android`
App Name: `Steam Mobile`
About: `Buy games, protect your Steam account, and get the latest news from your games and the community.`
